net-p2p/qbittorrent/qbittorrent-3.3.3.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils qt4-r2 flag-o-matic
```

Comment:
@RobinDX, qt4-r2 is deprecated and is succeeded by qmake-utils. Maybe consider porting to EAPI=6 and qmake-utils?